openlabs_report_webkit/__init__.py

```diff
@@ -29,56 +24,45 @@
 from trytond.tools import file_open
 from trytond.pool import Pool
 from trytond.transaction import Transaction
-from trytond.report import Report, TranslateFactory, Translator, FORMAT2EXT
+from trytond.report import Report, TranslateFactory, Translator
 from executor import execute
 
 
 class ReportWebkit(Report):
+    render_method = "webkit"
```

Comment:
Not sure if this is something that should be in code and if it is in code, it may even be better to depend on weasyprint (rather than webkit) for which we have no control over installation. Perhaps configuration ? Would each report depend on different engines is also another question.